ui/cypress.config.js

```diff
@@ -1,6 +1,8 @@
 const { defineConfig } = require("cypress");
 const webpackConfig = require('./cypress/webpack.cypress.js');
 
+const BASE_URL = process.env.CYPRESS_BASE_URL || "http://localhost:8080";
```

Comment:
This usually occurs all implicitly on cypress, but this allows to configure the base_url when not in a test context. For example, plugins/index runs on the nodejs cypress engine outside of a test context, and would also benefit from this env var being present.